static/js/containers/LearnerPage_test.js

```diff
@@ -59,9 +59,14 @@ import {
   clearSelectField,
   GoogleMapsStub,
 } from '../util/test_utils';
+import StaffLearnerInfoCard from '../components/StaffLearnerInfoCard';
+import {
+  REQUEST_DASHBOARD,
+  RECEIVE_DASHBOARD_SUCCESS,
+} from '../actions/dashboard';
 
 describe("LearnerPage", function() {
-  this.timeout(10000);
+  this.timeout(15000);
```

Comment:
We don't have to solve this here, but is there something in particular that is causing our tests here to run so long or is the LearnerPage just sufficiently complicated now that it takes this long to execute?